app/views/physical_server/show.html.haml

```diff
@@ -1,5 +1,7 @@
 #main_div
   - if %w(physical_servers).include?(@display)
     = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
+  - elsif @showtype == "main"
```

Comment:
We probably don't need this `elsif`, since the `"main"` case will be handled in `else`